_rules/SC4-1-2-iframe-unique-name.md

```diff
@@ -0,0 +1,205 @@
+</iframe>
+```
+
+#### Pass example 4
+
+Multiple `iframe` elements in the document having different `aria-labelledby` descriptions as accessible name.
+
+```html
+<div id="desc-for-title">List of Contributors</div>
+<iframe aria-labelledby="desc-for-title" src="../test-assets/SC4-1-2-iframe-unique-name-doc1.html">
+</iframe>
+<div id="desc-for-title1">List of Reviewers</div>
+<iframe aria-labelledby="desc-for-title1" src="../test-assets/SC4-1-2-iframe-unique-name-doc2.html">
+</iframe>
+```
+
+#### Pass example 5
+
+`frames` having the same `title` within a given document context, but one of them is not exposed to assistive technologies.
+
+```html
+<iframe style="display:none;" title="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc1.html">
+</iframe>
+<iframe title="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc2.html">
+</iframe>
+```
+
+
+#### Pass example 6
+
+`iframes` are allowed to have the same `title` across different document contexts. In this example `iframe#level2-frame1` has a parent document content of `iframe#level1-frame2`, and does not share root `level1` document context.
+
+```html
+<iframe id="level1-frame1" title="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc1.html">
+</iframe>
+<iframe id="level1-frame2" title="List of Contributors 2" src="../test-assets/SC4-1-2-iframe-unique-name-doc2.html">
+  <iframe id="level2-frame1" title="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc1.html">
+  </iframe>
+</iframe>
+```
+
+### Failed
+
+#### Fail example 1
+
+Multiple frames having the same `title` within a given document context.
+
+```html
+<iframe title="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc1.html">
+</iframe>
+<iframe title="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc2.html">
+</iframe>
+```
+
+#### Fail example 2
+
+Multiple frames having the same `aria-label` within a given document context.
+
+```html
+<iframe aria-label="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc1.html">
+</iframe>
+<iframe aria-label="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc2.html">
+</iframe>
+```
+
+#### Fail example 3
+
+Multiple frames having the same name given by `title` and `aria-label` within a given document context
+
+```html
+<iframe title="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc1.html">
+</iframe>
+<iframe aria-label="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc2.html">
+</iframe>
+```
+
+### Inapplicable
+
+#### Inapplicable example 1
+
+Accessible name provided is empty.
+
+```html
+<iframe title="" src="../test-assets/SC4-1-2-iframe-unique-name-doc1.html">
+</iframe>
+<iframe aria-label="" src="../test-assets/SC4-1-2-iframe-unique-name-doc3.html">
+</iframe>
+```
+
+#### Inapplicable example 2
+
+Usage of `alt` attribute to describe content is not valid.
```

Comment:
Can we make it more specific about why it is not valid, e.g. "`alt` cannot be used to provide accessible name for `iframe`"